contracts/script/HelloWorldDeployer.s.sol

```diff
@@ -19,18 +25,23 @@ contract HelloWorldDeployer is Script {
 
     address private deployer;
     address proxyAdmin;
+    StrategyBase helloWorldStrategy;
+    StrategyBase helloWorldStrategyImpl;
     CoreDeploymentLib.DeploymentData coreDeployment;
     HelloWorldDeploymentLib.DeploymentData helloWorldDeployment;
     Quorum internal quorum;
-
+    ERC20Mock token;
     function setUp() public virtual {
         deployer = vm.rememberKey(vm.envUint("PRIVATE_KEY"));
         vm.label(deployer, "Deployer");
 
         coreDeployment = CoreDeploymentLib.readDeploymentJson("deployments/core/", block.chainid);
+       
+        token = new ERC20Mock();
```

Comment:
Can you add the deployment of the mock token and the strategy to the DeploymentData struct in the HelloworldDeploymentLib

Reply:
And then add these to the json that gets written for the deployment

Reply:
ah good catch, changing this

Reply:
Currently the strategy is getting written to Json in run() but not the token - adding the token